ArduCopter/AP_Arming.cpp

```diff
@@ -96,15 +96,10 @@ bool AP_Arming_Copter::rc_calibration_checks(bool display_failure)
 
 bool AP_Arming_Copter::barometer_checks(bool display_failure)
 {
+    bool ret = AP_Arming::barometer_checks(display_failure);
```

Comment:
Any reason to not return false immediately? I think that's how it is done in the other places.